extensions/job_management/standard/requirements/job-management/create/REQ_response-body.adoc

```diff
@@ -0,0 +1,8 @@
+[[req_job-management_create_response-body]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/create-response-body
+part:: The response SHALL include a body that contains a status information of the created job that conforms to the https://schemas.opengis.net/ogcapi/processes/part1/1.0/openapi/schemas/statusInfo.yaml[statusInfo.yaml] schema.
+part:: The response SHALL contain a `created` status code and the `jobId` property that contains the job identifier.
```

Comment:
This is currently not quite aligned with openEO. There we just return a 201 with a Location header that points to GET /jobs/:id, which then includes created, id (not jobId, I thought we settled on id?), etc. Is that allowed, too?

Reply:
This is an alignment with OGC API `/processes/{processId}/execution` that returns minimally the status and job ID to make the response consistent with either endpoint. 

The `Location` should be mentioned. It is also used by OAP: https://docs.ogc.org/is/18-062r2/18-062r2.html#req_core_process-execute-success-async

And yes, regarding the `jobId`/`id` naming.

Reply:
Discard my comment about `Location`, it is already in the other requirement.